Asterix/tests/test_testbed.py

```diff
@@ -9,18 +10,23 @@
 
 def test_def_thd():
 
-    parameter_file_path = os.path.join(Asterix_root, 'Example_param_file.ini')
+    parameter_file_test = os.path.join(Asterix_root, 'tests', "param_file_tests.ini")
+    test_dir = get_data_dir(datadir="asterix_test_dir")
 
-    model_local_dir = os.path.join(get_data_dir(), "Model_local")
+    model_local_dir = os.path.join(test_dir, "Model_local")
 
     # Load configuration file
-    config = read_parameter_file(parameter_file_path)
+    config = read_parameter_file(parameter_file_test)
```

Comment:
I think is one of the instances where you could indeed do:
```python
def test_def_thd(tmpdir):
    ...
    model_local_dir = os.path.join(tmpdir)
```
and just remove the `test_dir = get_data_dir(datadir="asterix_test_dir")`.